### ISSUE TITLE
Build issues on OS X 10.15

### ISSUE THREAD

**Issue by anadi198** [nickgillian/grt#170]
> error:
>       out-of-line definition of 'getMinChange' does not match any declaration in
>       'GRT::MLBase'
> Float MLBase::getMinChange() const{
> error:
>       out-of-line definition of 'getRandomiseTrainingOrder' does not match any
>       declaration in 'GRT::MLBase'
> bool MLBase::getRandomiseTrainingOrder() const { return randomiseTrainin...

In a recent merge, `bool getUseValidationSet() const;` and `bool getRandomiseTrainingOrder() const;` were commented out inside GRT/CoreModules/MLBase.h. 

Uncommenting them removed the make errors.

Was getting errors from fstream inside examples, so had to remove the whole directory for now(since I have to get this working for a project asap). That helped with the make continuing further.

Then finally got errors from all the cpp files inside tools for the `#include <GRT/GRT.h>`.
Replaced all with `#include "../GRT/GRT.h"` and that fixed the issue. But I think there was a warning above those includes for this anyway.

**Comment by andymule** [comment]
this is right. I uncommented like above. Instead of removing directory (?) i disabled the examples from being built in my cmake options, and replaced all includes like above too. Thanks for the pointers!
